src/webview/webview2/mod.rs

```diff
@@ -727,41 +728,47 @@ window.addEventListener('mousemove', (e) => window.chrome.webview.postMessage('_
       _uidsubclass: usize,
       dwrefdata: usize,
     ) -> LRESULT {
+      type DataTuple = *mut (ICoreWebView2Controller, Rc<Window>);
       match msg {
         win32wm::WM_SIZE => {
-          let controller = dwrefdata as *mut ICoreWebView2Controller;
+          let window = &(*(dwrefdata as DataTuple)).1;
+          // Ignore resizing borderless wnidows when minimizing
+          if !window.is_decorated() && wparam == WPARAM(win32wm::SIZE_MINIMIZED as _) {
+            return DefSubclassProc(hwnd, msg, wparam, lparam);
+          }
+          let controller = &(*(dwrefdata as DataTuple)).0;
           let mut client_rect = RECT::default();
           win32wm::GetClientRect(hwnd, &mut client_rect);
-          let _ = (*controller).SetBounds(RECT {
+          let _ = controller.SetBounds(RECT {
             left: 0,
             top: 0,
             right: client_rect.right - client_rect.left,
             bottom: client_rect.bottom - client_rect.top,
           });
 
           if wparam == WPARAM(win32wm::SIZE_MINIMIZED as _) {
-            let _ = (*controller).SetIsVisible(false);
+            let _ = controller.SetIsVisible(false);
           }
 
           if wparam == WPARAM(win32wm::SIZE_RESTORED as _)
             || wparam == WPARAM(win32wm::SIZE_MAXIMIZED as _)
           {
-            let _ = (*controller).SetIsVisible(true);
+            let _ = controller.SetIsVisible(true);
           }
```

Comment:
just remove the visibility handling, and it should be good. This doesn't need to be only for frameless windows

Reply:
Done